rush-plugins/rush-resolver-cache-plugin/src/computeResolverCacheFromLockfileAsync.ts

```diff
@@ -74,7 +74,7 @@ function extractBundledDependencies(
 
     if (!foundBundledDependencies) {
       foundBundledDependencies = true;
-      nestedPackageDirs = nestedPackageDirs.slice(0);
+      context.nestedPackageDirs = nestedPackageDirs = nestedPackageDirs.slice(0);
```

Comment:
Add a comment here explicitly calling out the clone. Good to keep that context around.